### ISSUE TITLE
Fields of type "address" are not supported

### ISSUE THREAD

**Issue by urig** [QuickbaseAdmirer/QuickBase-C-Sharp-SDK#8]
### Steps to reproduce:
1. Given a QB App with a Table that has a Field of type "address",
2. Login to QB and connect to the App like so:
```
    var client = Intuit.QuickBase.Client.QuickBase.Login("username", "password", "domain");
    var app = client.Connect("app_id");
 ```

### Expected behavior:
The variable `app` should contain a valid instance of `IQApplication`.

### Actual behavior:
`ArgumentException` is thrown from `Intuit.QuickBase.Client.QTable.LoadColumns()` ([line 305](https://github.com/QuickbaseAdmirer/QuickBase-C-Sharp-SDK/blob/master/Intuit.QuickBase.Client/QTable.cs#L305))

### Root cause:
The exception is thrown when line 305 hits a field where `columnNode.GetAttribute("field_type", String.Empty)` is "address". The enum `FieldType` has no corresponding value so the exception is thrown.

**Comment by wmarrone** [comment]
I had the same issue when trying to implement in our Business.  Any fixes?

**Comment by jherig** [comment]
This repo hasn't been updated in 5 years.  I have attempted to correct several problems and make programming easier in [my fork](https://github.com/jherig/QuickBase-C-Sharp-SDK) and would appreciate feedback in what I've been doing.

**Comment by wmarrone** [comment]
That's great, I will watch your fork and try to work with it. My company is
trying to bring their month end numbers down into SQL and I was going to
setup something with the SDK to automatically take care of it at the end of
the month. Although I am having issues with the size of one of the tables
throwing a ViewTooLargeException. And I cannot seem to figure out how the
SDK is pulling the records down in the query, for instance the field IDs on
Quickbase never match the Table array returned.

**Comment by jherig** [comment]
I do have a fix for the ViewTooLarge in my fork, although it does have to result to a 'fallback and split query' which adds quite a few extra API calls and time... unfortunately the API doesn't allow any way to guess when a ViewTooLarge will occur.  Let me know if I can help.

**Comment by wmarrone** [comment]
Yeah the main problem I am having with it is when it queries it does not
return values in any structured way and there is way to get the structure
of the table return. Thankfully all we are trying to do is move the data
from Quickbase to an SQL server so if I could just get the structure moving
it would be easy. I do not know how the split would affect it though.

**Comment by jherig** [comment]
Perhaps I wasn't quite clear... my code does stitch the subqueries back together.  It just takes a lot longer that in should, and uses quite a few extra API calls which can matter depending on your QB subscription.

**Comment by wmarrone** [comment]
Got it, that shouldn't be a problem for us since I believe we have 250000 a
month and I'm the only real developer here so we shouldn't have a problem
with that.

**Comment by Shannonb1** [comment]
getting this multitext error using your fork

var type =
                    (FieldType) Enum.Parse(typeof(FieldType), columnNode.Attribute("field_type").Value, true);

the error
System.ArgumentException: 'Requested value 'multitext' was not found.'

**Comment by jherig** [comment]
Ah, thanks for reminding me about handling that field type. I don't recall the reason I put it off, but I think it had something to do with problems writing a test case for it. I'll see what I can do this week to at least support fields created via the webGUI.

**Comment by Shannonb1** [comment]
I added this to fieldtype.cs to solve


        /// <summary>
        /// Text-Multi-Line
        /// </summary>
        multitext
